src/main/java/edu/kis/powp/jobs2d/command/CommandImporter.java

```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.io.IOException;
+
+public interface CommandImporter {
+    DriverCommand importCommands(String filePath);
```

Comment:
I think that filePath is a wrong abstraction. Commands are imported directly from text strings. These text strings can be read from files, textboxes, url's, etc.